File: lib/core/models/question.dart
class QuestionModel {
  String? question, wAnswerOne, wAnswerTwo, wAnswerThree, correctAnswer, url;
  int? order;
  QuestionModel({
    required this.url,
    required this.question,
    required this.wAnswerOne,
    required this.wAnswerTwo,
    required this.wAnswerThree,
    required this.correctAnswer,
    required this.order,
  });

  factory QuestionModel.fromJson(Map<String, dynamic> json) => QuestionModel(
      question: json['question'],
      wAnswerOne: json['wAnswerOne'],
      wAnswerTwo: json['wAnswerTwo'],
      wAnswerThree: json['wAnswerThree'],
      correctAnswer: json['correctAnswer'],
      url: json['url'],
      order: json['order']);

  Map<String, String> toJson() {
    return {
      'question': question!,
      'wAnswerOne': wAnswerOne!,
      'wAnswerTwo': wAnswerTwo!,
      'wAnswerThree': wAnswerThree!,
      'correctAnswer': correctAnswer!,
      'url': url!,
      'order': order!.toString()
    };
  }
}

File: server/src/db/Question.js
const mongoose = require('mongoose');
const questionSchema = new mongoose.Schema({
    url:{
        type:String,
        required:false
    },
    question:{
        type:String,
        required:true
    },
    wAnswerOne:{
        type:String,
        required:true
    },
    wAnswerTwo:{
        type:String,
        required:true
    },
    wAnswerThree:{
        type:String,
        required:true
    },
    correctAnswer:{
        type:String,
        required:true
    },
    order:{
        type:Number,
        required:true
    },
    examUid:{
        type:String,
        required:true
    }
})

module.exports = mongoose.model('question',questionSchema);